src/GraphQL/GraphQLExtensions.cs

```diff
@@ -319,7 +318,11 @@ public static IValue AstFromValue(this object value, ISchema schema, IGraphType
 
                 if (!(value is string) && value is IEnumerable list)
                 {
-                    var values = list.Map(item => AstFromValue(item, schema, itemType));
+                    var values = list
+                        .Cast<object>()
+                        .Select(item => AstFromValue(item, schema, itemType))
+                        .ToList();
```

Comment:
Why do you use `.ToList()` here? Guessing from the old usage, it seems to me that `IEnumerable<T>` would be fine.

Reply:
The `Map()` extension method was calling `.ToList()` but then returning the list as `IEnumerable<T>`, so this replicates that behavior. And I'm not sure what's going to consume the enumerable later, so I want to make sure it doesn't get generated multiple times.